docs/config.md

```diff
@@ -5,15 +5,27 @@ These are the config settings available to jj/Jujutsu.
 
 ## Config files and TOML
 
-The config settings are loaded from the following locations. Less common ways to
-specify `jj` config settings are discussed in a later section.
+`jj` loads several types of config settings:
 
-* [The user config file]
-* `.jj/repo/config.toml` (per-repository)
+- The default settings. These cannot be edited.
```

Comment:
Nit: rather than default, shouldn't they be called the builtin settings? Also provide a link to them.

Reply:
Yes, "built-in" is better, thank you!

It's difficult to provide the link to the right version of the detailed settings. We could add them to the versioned docs similarly to the cli reference, but I am not sure I'll put the energy into that right now. It, we could insert a link to the prepare version with an awkward explanation.

I'm thinking of maybe just saying `they are located in the `cli/stc/config/` dir of the source tree.

Reply:
> It's difficult to provide the link to the right version of the detailed settings. We could add them to the versioned docs similarly to the cli reference, but I am not sure I'll put the energy into that right now. 

Totally fair. But we should do it at some point in the future.

> I'm thinking of maybe just saying `they are located in the `cli/stc/config/` dir of the source tree.

Yes, that pointer should be enough for those who are really curious.